dali/base/dacsds.cpp

```diff
@@ -1038,6 +1038,12 @@ bool CClientRemoteTree::isCompressed(const char *xpath) const
     return CRemoteTreeBase::isCompressed(xpath);
 }
 
+CompressionMethod CClientRemoteTree::getCompressionType() const
+{
+    checkExt();
```

Comment:
it may not matter, but if this tree has a COMPRESS_ATTR, it can return true, without calling checkExt()